File-Formats/DocIO/Working-with-Tables.md

```diff
@@ -2247,6 +2247,370 @@ document.Close();
+    // Apply built in table style as base style for CustomStyle1.
+    tableStyle.ApplyBaseStyle(BuiltinTableStyle.TableContemporary);
+    //Applies the custom table style to the table
+    table.ApplyStyle("CustomStyle1");
+
+    document.LastSection.AddParagraph();
+    //Create another table in the Word document.
+    table = document.LastSection.AddTable() as WTable;
+    table.ResetCells(3, 2);
+    table[0, 0].AddParagraph().AppendText("Row 1 Cell 1");
+    table[0, 1].AddParagraph().AppendText("Row 1 Cell 2");
+    table[1, 0].AddParagraph().AppendText("Row 2 Cell 1");
+    table[1, 1].AddParagraph().AppendText("Row 2 Cell 2");
+    table[2, 0].AddParagraph().AppendText("Row 3 Cell 1");
+    table[2, 1].AddParagraph().AppendText("Row 3 Cell2");
+
+    //Adds a new custom table style.
+    tableStyle = document.AddTableStyle("CustomStyle2") as WTableStyle;
+    tableStyle.TableProperties.RowStripe = 1;
+    //Apply conditional formatting for first row.
+    firstRowStyle = tableStyle.ConditionalFormattingStyles.Add(ConditionalFormattingType.FirstRow);
+    firstRowStyle.ParagraphFormat.HorizontalAlignment = HorizontalAlignment.Center;
+    //Apply conditional formatting for odd row.
+    oddRowBandingStyle = tableStyle.ConditionalFormattingStyles.Add(ConditionalFormattingType.OddRowBanding);
+    oddRowBandingStyle.CharacterFormat.TextColor = Color.Red;
+
+    //Add a new custom table style.
+    WTableStyle tableStyle2 = document.AddTableStyle("CustomStyle3") as WTableStyle;
+    tableStyle2.TableProperties.RowStripe = 1;
+    //Apply conditional formatting for first row.
+    ConditionalFormattingStyle firstRowStyle2 = tableStyle2.ConditionalFormattingStyles.Add(ConditionalFormattingType.FirstRow);
+    firstRowStyle2.CellProperties.BackColor = Color.Blue;
+    //Apply conditional formatting for odd row.
+    ConditionalFormattingStyle oddRowStyle2 = tableStyle2.ConditionalFormattingStyles.Add(ConditionalFormattingType.OddRowBanding);
+    oddRowStyle2.CellProperties.BackColor = Color.Yellow;
+    //Apply custom table style as base style for another custom table style.
+    tableStyle2.ApplyBaseStyle("CustomStyle2");
+    //Apply the custom table style to the table.
+    table.ApplyStyle("CustomStyle3");
+
+    //Save the Word document to the MemoryStream.
+    MemoryStream outputStream = new MemoryStream();
+    document.Save(outputStream, FormatType.Docx);
+    //Save the stream as a file in the device and invoke it for viewing.
+    Xamarin.Forms.DependencyService.Get<ISave>().SaveAndView("Sample.docx", "application/msword", outputStream);
+    //Please download the helper files from the below link to save the stream as file and open the file for viewing in Xamarin platform
+    //https://help.syncfusion.com/file-formats/docio/create-word-document-in-xamarin#helper-files-for-xamarin
+}
+{% endhighlight %}
+
+{% endtabs %}
+
+You can download a complete working sample from [GitHub]().
```

Comment:
We will add this line later, once sample merged in live. At now, remove this line

Reply:
@MohanaselvamJothi -I have removed it